bundles/binding/org.openhab.binding.zwave/database/cooper/rfwc5.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+	<Model>RFWC5</Model>
+	<Label lang="en">Aspire RF Wall Mount Controller 5-Scene Keypad
+	</Label>
+	<CommandClasses>
+		<Class>
+			<id>0x00</id>
+		</Class>
+		<Class>
+			<id>0x20</id>
+		</Class>
+		<Class>
+			<id>0x21</id>
+		</Class>
+		<Class>
+			<id>0x22</id>
+		</Class>
+		<Class>
+			<id>0x2B</id>
+		</Class>
+		<Class>
+			<id>0x2D</id>
+		</Class>
+		<Class>
+			<id>0x70</id>
+		</Class>
+		<Class>
+			<id>0x72</id>
+		</Class>
+		<Class>
+			<id>0x77</id>
+		</Class>
+		<Class>
+			<id>0x85</id>
+		</Class>
+		<Class>
+			<id>0x86</id>
+		</Class>
+		<Class>
+			<id>0x87</id>
+		</Class>
+	</CommandClasses>
+	<Associations>
```

Comment:
Are any of these association groups in use by the controller or are they just used to control other devices?  If they need to be configured to communicate with the controller, then you need to add ```<SetToController>true</SetToController>``` for the appropriate groups...